README.md

```diff
@@ -71,7 +71,43 @@ Required input:
 </p>    
 </div>
 
-### Resources
+## <div align="center">High level features and techniques</div>
+
+MCT offers a range of powerful features to optimize models for efficient edge deployment. These supported features include:
+
+### Quantization Core Features
+
+🏆 **Mixed-precision search** [![Open In Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/sony/model_optimization/blob/main/tutorials/notebooks/mct_features_notebooks/pytorch/example_pytorch_mixed_precision_ptq.ipynb). Assigning optimal quantization bit-width per layer (for weights/activations) 
+
+📈 **Graph optimizations**. Transforming the model to be best fitted for quantization process.
+
+🔎 **Quantization parameter search**. Minimizing expected quantization-noise during thresholds search using methods such as MSE, No-Clipping and MAE.
+
+🧮 **Advanced quantization algorithms**. To prevent a performance degradation some algorithms are applied, such as Shift negative correction, Outliers filtering and clustering.
+__________________________________________________________________________________________________________
+### Hardware-aware optimization 
+
+🎯 **TPC (Target Platform Capabilities)**. Describes the target hardware’s constrains, for which the model optimization is targeted. See [TPC Readme](https://github.com/sony/model_optimization/blob/main/model_compression_toolkit/target_platform_capabilities/README.md) for more information.
```

Comment:
change in-repo link to relative. here for example it should be: ./model_compression_toolkit/target_platform_capabilities/README.md